README.md

```diff
@@ -243,12 +243,18 @@ Some nice to have but probably separate items:
 * [kwasm](https://github.com/jasonwyatt/KWasm)
 * [wazero](https://wazero.io/)
 
-### Building Locally
+### Building the Runtime
+
+Contributors and other advanced users may want to build the runtime from source. To do so, you'll need to have Maven installed.
+
+Basic steps:
 
 * `mvn clean install` to run all of the project's tests and install the library in your local repo
 * `mvn spotless:apply` to autoformat the code
 * `./scripts/compile-resources.sh` will recompile and regenerate the `resources/compiled` folders
 
+NOTE: The `install` target relies on the `wabt` library to compile the test suite. This is not currently released for ARM (e.g. new Macs with Apple Silicon). However, `wabt` is available from Homebrew, so `brew install wabt` before running `mvn clean install` should work.
```

Comment:
Note that the mvn plugin should download this for you. So that might be failing?